baselines/webworker.generated.d.ts

```diff
@@ -383,7 +388,7 @@ interface IDBObjectStore {
     deleteIndex(indexName: string): void;
     get(key: any): IDBRequest;
     index(name: string): IDBIndex;
-    openCursor(range?: any, direction?: string): IDBRequest;
+    openCursor(range?: IDBKeyRange | IDBValidKey, direction?: string): IDBRequest;
     put(value: any, key?: any): IDBRequest;
```

Comment:
also should be `put(value: any, key?: IDBValidKey): IDBRequest;`. Other methods are same.